packages/client3/components/design-system/logo/logo.tsx

```diff
@@ -1,16 +1,56 @@
+import Image from "next/image";
+
+import FreeArtistsFullLogo from "/public/FAD-logo-full.png";
+
 import freeArtistsLogoWhiteBgPng from "./FAD-logo-white-bg.png";
 import freeArtistsLogoPng from "./FAD-logo.png";
 import FreeArtistsLogoSvg from "./FAD-logo.svg";
 
-export const freeArtistsLogoPngUrl = freeArtistsLogoPng.src;
-export const freeArtistsLogoWhiteBgPngUrl = freeArtistsLogoWhiteBgPng.src;
+export const freeArtistsLogoPngUrl = "/images/fa-logo.png";
+export const freeArtistsLogoWhiteBgPngUrl = "/public/fa-logo-white-bg.png";
+export const freeArtistsFullLogoPngFull = FreeArtistsFullLogo.src;
 
 interface FreeArtistsLogoProps {
   width?: number;
   height?: number;
   className?: string;
+  type?: FreeArtistsLogoType;
+  alt?: string;
+  src?: string;
+  storyBookMode?: string;
+}
+
+enum FreeArtistsLogoType {
+  pngWhiteBg,
+  png,
+  pngFull,
 }
 
 export function FreeArtistsLogo(props: FreeArtistsLogoProps) {
   return <FreeArtistsLogoSvg {...props} />;
 }
+
+export function FreeArtistsLogoFull(props: FreeArtistsLogoProps) {
+  const src = props.src;
+  const freeArtistsFull = "http://localhost:3001/FAD-logo-full.png";
```

Comment:
I wouldn't suggest putting up an absolute path in code. I think it's added to serve images in Next Image but I believe a relative path could be used instead.